lexer.js

```diff
@@ -1320,3 +1320,8 @@ function isExpressionTerminator (curPos) {
   }
   return false;
 }
+
+const initPromise = Promise.resolve();
+
+module.exports.init = () => initPromise;
+module.exports.parse = parseCJS;
```

Comment:
Note that this could be done without breaking changes:

```js
module.exports = (source, name) => parseCJS(source, name);
module.exports.init = () => initPromise;
module.exports.parse = parseCJS;
```

Reply:
good point. @guybedford thoughts?

Reply:
Let's keep it simple as a major change actually. Saves on type differences / leaving deprecation for subsequent work.